reference/tax_provider.yml

```
@@ -250,13 +250,13 @@ paths:
         '400':
           description: Fallback Tax will be used for this transaction. General response that points to an issue with the incoming request that means a valid response is unable to be returned.
         '401':
-          description: Response to indicate that the merchant's authentication credentials are invalid. The merchant will receive an update in their Store Logs.
+          description: Response to indicate that the merchant’s authentication credentials are invalid. The merchant will receive an update in their Store Logs.
         '500':
           description: Fallback Tax will be used for this transaction. General response that points to an error on the tax provider side. These types of errors should be promptly resolved by the tax provider.
   /void:
     post:
       summary: Void Tax Quote
-      description: Invalidate the persisted tax quote as identified by the given unique ID. Relevant to order cancellations or when moving an order from a paid status to an unpaid status.
+      description: Invalidate the persisted tax quote as identified by the given unique ID. Relevant to order cancellations or when moving an order from a paid status to unpaid status.
```

```suggestion
      description: Invalidate the persisted tax quote as identified by the given unique ID. Relevant to order cancellations or when moving an order from a paid status to an unpaid status.
```
revert Traci's suggestion